security/default-backend/ndn-lite-default-rng-impl.c

```diff
@@ -8,13 +8,16 @@
  * See AUTHORS.md for complete list of NDN-LITE authors and contributors.
  */
 
+#include <stddef.h>
 #include "ndn-lite-default-rng-impl.h"
 #include "../ndn-lite-rng.h"
 #include "../../ndn-error-code.h"
 
 /* always fails and return 0 */
 static int ndn_lite_default_rng(uint8_t *dest, unsigned size)
 {
+    if (dest == NULL || size == 0)
+      return 0;
```

Comment:
This condition does make sense.
Just `(void*)` unused parameters.

Reply:
Fixed with the latest commit (9c74035)